### PR TITLE
PLF-8674: Disabled users' display issues

### PR DESCRIPTION
**Pull request by lbengayed**

### CODE REVIEW

File: component/service/pom.xml [hunk 1]
```diff
@@ -295,6 +295,10 @@
         <artifactId>jersey-client</artifactId>
         <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.exoplatform.social</groupId>
+          <artifactId>social-component-webui</artifactId>
```
Comment (boubaker) [inline component/service/pom.xml]: The Rest Services shouldn't depend on Webui artifact

File: component/service/src/main/java/org/exoplatform/social/service/rest/PeopleRestService.java [hunk 2]
```diff
@@ -499,7 +499,10 @@ public Response suggestUsernames(@Context UriInfo uriInfo,
   private LinkedHashSet<UserInfo> addSpaceMembers (String spaceURL, ProfileFilter identityFilter, LinkedHashSet<UserInfo> userInfos, String currentUser) {
     String[] spaceMembers = getSpaceService().getSpaceByUrl(spaceURL).getMembers();
     for (String spaceMember : spaceMembers) {
-      userInfos = addUsernameToInfosList(spaceMember, identityFilter,userInfos, currentUser, true);
+      Identity identity = LinkProvider.getIdentityManager().getOrCreateIdentity(OrganizationIdentityProvider.NAME, spaceMember, false);
```
Comment (ahamdi) [inline component/service/src/main/java/org/exoplatform/social/service/rest/PeopleRestService.java]: Why not 

```suggestion
      Identity identity = getIdentityManager().getOrCreateIdentity(OrganizationIdentityProvider.NAME, spaceMember, false);

```

**Review by ahamdi — CHANGES_REQUESTED** [review]
Please update Unit tests in PeopleRestServiceTest

**Review by ahamdi — APPROVED** [review]
LGTM

File: component/service/pom.xml (hunk 1 above)
Comment (ahamdi) [reply component/service/pom.xml]: @boubaker  This was fixed, could you re-validate please :)

**Pull request merged**